app/models/Date.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
I wonder if it might be worth looking into a PHP library for doing date and time handling?
The library Chrono is pretty neat

Reply:
Ye that could be good too, i like to do it this way first though